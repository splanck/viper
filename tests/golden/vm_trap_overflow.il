il 0.1.2
func @main() -> i64 {
entry:
  .loc 1 1 1
  %t0:i64 = iadd.ovf 9223372036854775807, 1
  ret %t0
}
