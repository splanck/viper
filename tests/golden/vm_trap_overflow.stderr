Trap @main#0 line 1: Overflow \(code=0\)
